frontend-react/src/components/header/DropdownNav.tsx

```diff
@@ -40,7 +40,7 @@ export const DropdownNav = ({ label, root, directories }: DropdownNavProps) => {
         };
     }, []);
     const navMenu = directories.map((dir) => (
-        <NavLink to={`${dir.root}/${dir.slug}`}>{dir.title}</NavLink>
+        <USLink href={`${dir.root}/${dir.slug}`}>{dir.title}</USLink>
```

Comment:
Furthermore, the sub-item of a dropdown is _not_ a `NavLink` because uswds does not have an active style for a chosen dropdown navigation item.